src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1003,13 +1003,15 @@ def auto_shutdown_vm(cmd, resource_group_name, vm_name, off=None, email=None, we
         raise CLIError('usage error: --time is a required parameter')
     daily_recurrence = {'time': time}
     notification_settings = None
-    if webhook:
+    if email:
```

Comment:
is this service behavior? If no, this will be breaking change to end users: adding a required param for email.

Reply:
Yeah, the webhook parameter is also required if the user needs to set --email. This issue is that the user doesn't know the settings -- email needs to be set -- webhook. I make this change to prompt the user that both parameters must exist at the same time for them to take effect

Reply:
Do we allow webhook and no email?